docs/Makefile.am

```diff
@@ -834,12 +834,12 @@ EXTRA_DIST += \
         $(OSHMEM_MAN1_FORTRAN_REDIRECTS)
 
 ###########################################################################
-if OPAL_BUILD_DOCS
+if OAC_BUILD_DOCS
```

Comment:
I should have looked at this patch before reviewing the OAC patch.  It would really be good to avoid using variables that start with `OAC_`, so that we can use the Autoconf feature to blocklist `OAC_` and error if macros don't expand properly.

Reply:
Ah, are you referring to [m4_pattern_forbid](https://www.gnu.org/software/autoconf/manual/autoconf-2.60/html_node/Forbidden-Patterns.html)?

If so, should I add another argument to `OAC_SETUP_SPHINX` that's the prefix of the AM_CONDITIONAL names to create (i.e., `{prefix}_BUILD_DOCS` and `{prefix}_INSTALL_DOCS`)?